lib/mastodon/cli/accounts.rb

```diff
@@ -472,15 +472,13 @@ def reset_relationships(username)
       end
 
       total     = 0
-      total    += Account.where(id: ::Follow.where(account: account).select(:target_account_id)).count if options[:follows]
-      total    += Account.where(id: ::Follow.where(target_account: account).select(:account_id)).count if options[:followers]
+      total    += follows_for(account).count if options[:follows]
+      total    += followers_for(account).count if options[:followers]
```

Comment:
I'm wondering why this has been written this way in the first place, and shouldn't this just be `account.following` and `account.followers`?
Having a look, the SQL queries are slightly different, but the query plans are exactly the same.